src/Stack/Types/PackageDump.hs

```diff
@@ -31,4 +31,6 @@ data InstalledCacheEntry = InstalledCacheEntry
 instance Store InstalledCacheEntry
 
 installedCacheVC :: VersionConfig InstalledCacheInner
-installedCacheVC = storeVersionConfig "installed-v1" "GGyaE6qY9FOqeWtozuadKqS7_QM="
+-- FIXME would be really nice to avoid changing this hash...
+--installedCacheVC = storeVersionConfig "installed-v1" "GGyaE6qY9FOqeWtozuadKqS7_QM="
```

Comment:
That's unfortunate, I was hoping there was some way to add backwards-compatibility support using the old serialization format.

I understand the goal of being less dependent on the Cabal API, but I don't think it's too relevant here: we're already wrapping up these data types internally, and can easily wrap them differently in the future. The advantage is that Cabal's new representations are more space efficient (using unpinned memory and UTF-8 encoding, or in the case of `Version` encoding to a single `Word64`). But if we're going to lose caching, I'd rather just back out this change.